docs/Install.md

```diff
@@ -1,22 +1,125 @@
 
 # Installation
 
-The following will walk through the deployment of Pelorus.
+There are three methods to deploy Pelorus:
+
+(Recommended) First two are using Pelorus Operator that is available by default in the [Red Hat-provided community-operators catalog sources](https://docs.openshift.com/container-platform/4.11/operators/understanding/olm-rh-catalogs.html#olm-rh-catalogs_olm-rh-catalogs). This catalog is distributed as part of the OpenShift:
+
+  * OpenShift [Command Line Tool](#openshift-command-line-tool)
+  * OpenShift [Dashboard](#openshift-dashboard)
```

Comment:
Throughout my studies, I have seen the use of "web console" instead of "dashboards". Would be better to change?